src/askvm/resources/core/let.ts

```diff
@@ -47,9 +47,6 @@ export const letRes = resource({
 
     options.values![key] = value;
 
-    if (code.name === 'const') {
-      Object.freeze(options.values![key]);
-    }
```

Comment:
The part for `assign` in this file should also be removed

Reply:
Removed